src/common/index.js

```diff
@@ -1,10 +1,4 @@
 import * as utils from './utils';
 import PageLoading from './components/PageLoading';
-import { configureUserAccountApiService, fetchUserAccount } from './actions';
 
-export {
-  PageLoading,
-  utils,
-  configureUserAccountApiService,
```

Comment:
Why is this being removed?

Reply:
These aren't needed anymore as the `initialize()` function from frontend-platform takes care of fetching and authenticating the user's account.